pyuvdata/tests/__init__.py

```diff
@@ -143,13 +169,18 @@ def skip(reason):
     """
     Defines a decorator to unconditionally skip a test. Called by conditional
     skip wrappers to skip tests that require optional dependencies.
-
     This is needed because nose doesn't respect unittest skip_if decorators.
     Based on: https://stackoverflow.com/questions/21936292/conditional-skip-testcase-decorator-in-nosetests
     Args:
         reason: String describing the reason for skipping a test.
     """
+    warnings.simplefilter('always', DeprecationWarning)  # turn off filter
+    warnings.warn("The skip function was built to work with the `nose` test "
+                  "suite and is deprecated. It will be released in a future version.",
```

Comment:
"removed in a future version"?

Reply:
I could have sworn I wrote "removed in version 1.5". I blame the headache of the decorator that I originally tried to use